src/main/cms/cms.c

```diff
@@ -259,17 +275,69 @@ static void cmsPadToSize(char *buf, int size)
 
     buf[size] = 0;
 }
+#endif
+
+// Pad buffer to the left, i.e. align right
+static void cmsPadLeftToSize(char *buf, int size)
+{
+    int i,j;
+    int len = strlen(buf);
+
+    for (i = size-1, j = size-len ; i-j >= 0 ; i--) {
+      buf[i] = buf[i-j];
+    }
+
+    for ( ; i >= 0 ; i--) {
+      buf[i] = ' ';
+    }
+
+    buf[size] = 0;
+}
+
+static void cmsPadToSize(char *buf, int size)
+{
+  // Make absolutly sure the string terminated.
+  buf[size] = 0x00,
+
+#ifdef CMS_OSD_RIGHT_ALIGNED_VALUES
+  cmsPadLeftToSize(buf, size);
+#else
+  smallScreen ? cmsPadLeftToSize(buf, size) : cmsPadRightToSize(buf, size);
+#endif
+}
+
+static int cmsDrawMenuItemValue(displayPort_t *pDisplay, char *buff, uint8_t row, uint8_t maxSize)
+{
+  int colpos;
+  int cnt;
+
+  cmsPadToSize(buff, maxSize);
+#ifdef CMS_OSD_RIGHT_ALIGNED_VALUES
+  colpos = rightMenuColumn - (uint8_t)strlen(buff);
+#else
+  colpos = smallScreen ? rightMenuColumn - (uint8_t)strlen(buff) : rightMenuColumn;
+#endif
+  cnt = displayWrite(pDisplay, colpos, row, buff);
+  return cnt;
+}
```

Comment:
Looks like some of this is indented 2 spaces. Can you change it to 4?

Reply:
OK, fixed.